app/templates/components/builds-item.hbs

```diff
@@ -81,7 +81,7 @@
   </h3>
   <div class="row-item row-commit">
     <ExternalLinkTo @href={{this.build.commit.url}}>
-      <EmberTooltip @text="{{concat "View commit on " (vcs-name this.build.repo.vcsType)}}" />
+      <EmberTooltip @text="{{concat "View commit on " (vcs-name this.vcsType)}}" />
```

Comment:
It seems fine, though I'm not sure I understand the motivation for this change

Reply:
We reduced repo representation in response and we don't have vcs_type in json, so this must be passed from parent element

Reply:
ahh gotcha, makes sense :+1: Thanks!